libs/auth/src/reducers/auth-reducer-state.interface.ts

```diff
@@ -0,0 +1,7 @@
+import { DaffAuthToken } from '../models/auth-token';
+
+export interface DaffAuthReducerState<T extends DaffAuthToken> {
```

Comment:
I think this model is wrong. "login" and "register" probably have split "Loadable" states (errors + loading).

Reply:
that makes sense